episodes/03-create.md

```diff
@@ -36,38 +64,138 @@ Courtesy NASA/JPL-Caltech.
 
+If using RStudio desktop, the project is opened in a new session. Otherwise, the working directory and active project is changed:
+
+```r
+usethis::create_project(path = "cases")
+```
+
+```output
+✔ Creating 'cases/'
+✔ Setting active project to 'C:/~/cases'
+✔ Creating 'R/'
+✔ Writing 'cases.Rproj'
+✔ Adding '.Rproj.user' to '.gitignore'
+✔ Opening 'C:/~/cases/' in new RStudio session
+✔ Setting active project to '<no active project>'
+```
+
+<!--
 ```bash
 $ cd ~/Desktop
-$ mkdir planets
-$ cd planets
+$ mkdir cases
+$ cd cases
 ```
+-->
 
-Then we tell Git to make `planets` a [repository](../learners/reference.md#repository)
+Then we tell Git to make `cases` a [repository](../learners/reference.md#repository)
 \-- a place where Git can store versions of our files:
 
+```r
+usethis::use_git()
+```
+
+```output
+✔ Setting active project to 'C:/~/cases'
+✔ Initialising Git repo
+✔ Adding '.Rhistory', '.Rdata', '.httr-oauth', '.DS_Store', '.quarto' to '.gitignore'
+There are 2 uncommitted files:
+* '.gitignore'
+* 'cases.Rproj'
+Is it ok to commit them?
+```
+
+Remember that each record of change can be [commit](../learners/reference.md#commit). So, you can make these two files, `.gitignore` and `cases.Rproj`, part of it. Select that Yes, you agree!
+
+```output
+✔ Adding files
+✔ Making a commit with message 'Initial commit'
+• A restart of RStudio is required to activate the Git pane
+Restart now?
+```
+
+Agree to restart your session to activate the Git pane in Rstudio:
+
+![The Git tab in the Environments pane shows the `status` of your repository.](fig/pane-git-01.png)
+
+The Git tab is in the [Environments pane](https://docs.posit.co/ide/user/ide/guide/ui/ui-panes.html), usually in the upper right corner of the Rstudio IDE. 
+
+In this and next episodes you'll learn the function of all those buttons on the top of the Git tab!
+
+<!--
 ```bash
 $ git init
 ```
+-->
 
-It is important to note that `git init` will create a repository that
+It is important to note that `usethis::use_git()` will create a repository that
 can include subdirectories and their files---there is no need to create
-separate repositories nested within the `planets` repository, whether
+separate repositories nested within the `cases` repository, whether
 subdirectories are present from the beginning or added later. Also, note
-that the creation of the `planets` directory and its initialization as a
+that the creation of the `cases` directory and its initialization as a
 repository are completely separate processes.
 
-If we use `ls` to show the directory's contents,
-it appears that nothing has changed:
+This step is known as `git init` because it initialise your Git repository.
+
+::::::::::::::::: checklist
+
+### Checklist
+
+![Set up `Git` once per computer. Initialize `Git` once per project.](fig/git-rstudio-06.png)
```

Comment:
This image did not render for me. Check to see if the file path is correct.

Reply:
Was not committed. Included now in https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/3aab915e29395517e550fb9b63a369aec4f7431d